src/api/admin.ts

```diff
@@ -0,0 +1,77 @@
+import { DB, ServiceKey } from './types/admin'
+import express from 'express'
+import fs from 'fs'
+
+const app = express.Router()
+const data: DB = {
+  serviceKeys: []
+}
+
+const writeFile = ()=>{
+  const fileData: string = JSON.stringify(data)
+  fs.writeFile('./data.db', fileData,  (err)=>{
+    if(err) {
+      return console.log('error', err)
+    }
+    console.log('file write success')
+  })
+}
+
+fs.readFile('./data.db', (err, file)=>{
+  if(err){
+    return console.log('error', err)
+  }
+  console.log('file read success')
+  const fileData: DB = JSON.parse(file.toString() || '{}')
+  data.serviceKeys = fileData.serviceKeys || []
+})
+
+app.post('/serviceHostsKeys', (req, res)=>{
```

Comment:
please change all api to `serviceHostKeys`. serviceHost`s`Keys is incorrect for what this service is doing.